_examples/clone/auth/ssh/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+	"github.com/go-git/go-git/v5/plumbing/transport/ssh"
+	"os"
+
+	git "github.com/go-git/go-git/v5"
+	. "github.com/go-git/go-git/v5/_examples"
+)
+
+func main() {
+	CheckArgs("<url>", "<directory>", "<private_key_file>")
+	url, directory, privateKeyFile := os.Args[1], os.Args[2], os.Args[3]
+	var password string
+	if len(os.Args) == 5 {
+		password = os.Args[4]
+	}
+
+	_, err := os.Stat(privateKeyFile)
+	if err != nil {
+		Warning("read file %s failed %s\n", privateKeyFile, err.Error())
+		return
+	}
+
+	// Clone the given repository to the given directory
+	Info("git clone %s %s using ssh", url, directory)
```

Comment:
this should be the equivalent git command, not a comment

Reply:
ok

Reply:
resolved